JavaScript/JavaScriptSDK/Sender.ts

```diff
@@ -368,12 +401,20 @@ module Microsoft.ApplicationInsights {
         public _xhrReadyStateChange(xhr: XMLHttpRequest, payload: string[], countOfItemsInPayload: number) {
             if (xhr.readyState === 4) {
                 if ((xhr.status < 200 || xhr.status >= 300) && xhr.status !== 0) {
-                    this._onError(payload, xhr.responseText || xhr.response || "");
+                    if (!this._config.disableRetry() && this._isRetriable(xhr.status)) {
```

Comment:
I agree, but I was trying to keep it consistent with other configuration switches (.e.g disableFlushOnBeforeUnload, disableAjaxTracking or  enableSessionStorageBuffer).